File: app/src/main/java/com/yoesuv/infomadiun/fragment/ChildFragmentInfo.java
package com.yoesuv.infomadiun.fragment;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import com.yoesuv.infomadiun.BuildConfig;
import com.yoesuv.infomadiun.R;

public class ChildFragmentInfo extends Fragment {

    private View v;
    private TextView tvVersion;

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        v = inflater.inflate(R.layout.activity_info, container, false);

        tvVersion = (TextView) v.findViewById(R.id.textView_version_app);
        String ver = getResources().getString(R.string.versi)+" "+BuildConfig.VERSION_NAME;
        tvVersion.setText(ver);

        return v;
    }
}

File: app/src/main/java/com/yoesuv/infomadiun/menu/other/views/ChildFragmentInfo.kt
package com.yoesuv.infomadiun.menu.other.views

import android.content.Intent
import androidx.databinding.DataBindingUtil
import android.net.Uri
import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.yoesuv.infomadiun.BuildConfig
import com.yoesuv.infomadiun.R
import com.yoesuv.infomadiun.databinding.ChildFragmentInfoBinding

class ChildFragmentInfo : Fragment() {

    companion object {
        fun getInstance(): Fragment {
            return ChildFragmentInfo()
        }
    }

    private lateinit var binding: ChildFragmentInfoBinding

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
        binding = DataBindingUtil.inflate(inflater, R.layout.child_fragment_info, container, false)

        val version = resources.getString(R.string.info_app_version, BuildConfig.VERSION_NAME)
        binding.textViewVersion.text = version
        binding.textViewGithub.setOnClickListener {
            val intent = Intent(Intent.ACTION_VIEW, Uri.parse(getString(R.string.github_url)))
            context?.startActivity(intent)
        }

        return binding.root
    }

}